config.go

```diff
@@ -133,15 +136,25 @@ const (
 )
 
 var (
-	defaultLndDir     = btcutil.AppDataDir("lnd", false)
-	defaultConfigFile = filepath.Join(defaultLndDir, defaultConfigFilename)
-	defaultDataDir    = filepath.Join(defaultLndDir, defaultDataDirname)
-	defaultLogDir     = filepath.Join(defaultLndDir, defaultLogDirname)
+	// DefaultLndDir is the default directory where lnd tries to find its
+	// configuration file and store its data. This is a directory in the
+	// user's application data, for example:
+	//   C:\Users\<username>\AppData\Local\Lnd on Windows
+	//   ~/.lnd on Linux
+	//   ~/Library/Application Support/Lnd on MacOS
+	DefaultLndDir = btcutil.AppDataDir("lnd", false)
```

Comment:
Nice there're sooo many projects we have where this and many others have been copied over.